# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/internal_types.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/internal_types.h
--- a/security/sandbox/chromium/sandbox/win/src/internal_types.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/internal_types.h	Wed Jun 12 11:10:48 2019 +0100
@@ -34,9 +34,7 @@
 
   uint32_t Size() const { return size_; }
 
-  void* Buffer() const {
-    return buffer_;
-  }
+  void* Buffer() const { return buffer_; }
 
  private:
   uint32_t size_;
@@ -47,27 +45,21 @@
 // 32 and 64 bit builds. This construct is non-portable.
 class IPCInt {
  public:
-  explicit IPCInt(void* buffer) {
-    buffer_.vp = buffer;
-  }
+  explicit IPCInt(void* buffer) { buffer_.vp = buffer; }
 
-  explicit IPCInt(unsigned __int32 i32) {
-    buffer_.vp = NULL;
+  explicit IPCInt(uint32_t i32) {
+    buffer_.vp = nullptr;
     buffer_.i32 = i32;
   }
 
-  unsigned __int32 As32Bit() const {
-    return buffer_.i32;
-  }
+  uint32_t As32Bit() const { return buffer_.i32; }
 
-  void* AsVoidPtr() const {
-    return buffer_.vp;
-  }
+  void* AsVoidPtr() const { return buffer_.vp; }
 
  private:
   union U {
     void* vp;
-    unsigned __int32 i32;
+    uint32_t i32;
   } buffer_;
 };
 